src/scripts/main.js

```diff
@@ -1,3 +1,110 @@
 'use strict';
 
-// write code here
+class Table {
+  TBODY = document.querySelector('.field tbody');
```

Comment:
Avoid querying the DOM in the class field declaration. This is because the class may be instantiated before the DOM is fully loaded, resulting in these properties being `null`. It's better to move these queries into the `constructor` or an `init` method that's called after the DOM is fully loaded.